SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+set -ex
+
+parse_cmdline_args() {
+    # Ensure that the 'dracut-lib' is present and loaded.
+    type getarg >/dev/null 2>&1 || . /lib/dracut-lib.sh
+
+    VERITY_MOUNT="mnt/verity_mnt"
+    OVERLAY_MOUNT="/mnt/overlay_mnt"
```

Comment:
should mount folders be keyed off process id or some random value? I wonder if there might be a conflict with something else that uses these folders... unlikely, but could happen...

Reply:
I see, thanks, I have appended the process id, please check again.